lib/mergeusertool.php

```diff
@@ -37,7 +37,6 @@
 global $CFG;
 
 require_once $CFG->dirroot . '/lib/clilib.php';
-require_once __DIR__ . '/autoload.php';
```

Comment:
do not remove this line. It was here for some reason. Depending on the trace of using this class, it was necessary to have this here.

Reply:
ok